### ISSUE TITLE
MoE-LLaVA-StableLM for 4-bits and 8-bit

### ISSUE THREAD

**Issue by NikiBase** [PKU-YuanGroup/MoE-LLaVA#30]
Hello! 

I would like to ask if there are any plans to implement support with 4 and 8 bits to be able to infer with the model?

Thank you!

**Comment by LinB203** [comment]
Yes, but it seem that it conflicts with deepspeed, we still try to solve it.
